apps/src/codebridge/Workspace/HeaderButtons.tsx

```diff
@@ -45,6 +71,18 @@ const WorkspaceHeaderButtons: React.FunctionComponent = () => {
           />
         </WithTooltip>
       )}
+      {appName === 'pythonlab' && skipUrl && (
+        <Button
+          iconRight={{iconStyle: 'solid', iconName: 'arrow-right'}}
+          onClick={onClickSkip}
+          size={'xs'}
+          type={'tertiary'}
+          color={buttonColors.white}
+          text={commonI18n.skipToProject()}
```

Comment:
does this button have the same hover/active/disabled color as the other buttons in the header? We do some customization with [this class](https://github.com/code-dot-org/code-dot-org/blob/cb251751f7ef3457e5b38e1dc3212bcb1778971c/apps/src/lab2/styles/dark-mode.module.scss#L3-L13) that we may want to also apply here

Reply:
Slack discussion about renaming style class name to `tertiaryButton` in `dark-mode.module.scss` and applying this same rule. Thanks! https://codedotorg.slack.com/archives/C06JELCAPT9/p1731690038015439?thread_ts=1731536274.789579&cid=C06JELCAPT9